src/analysis/retail/priest/holy/Guide.tsx

```diff
@@ -39,7 +40,18 @@ function CooldownGraphSubsection({ modules, events, info }: GuideProps<typeof Co
       <strong>Cooldown Graph</strong> - this graph shows when you used your cooldowns and how long
       you waited to use them again. Grey segments show when the spell was available, yellow segments
       show when the spell was cooling down. Red segments highlight times when you could have fit a
-      whole extra use of the cooldown.
+      whole extra use of the cooldown.<br></br>
+      <br></br>
```

Comment:
rather than using `<br/>`, use either `<div>` or `<p>` elements wrapping the blocks of text.